src/pyforest/_imports.py

```diff
@@ -69,21 +69,40 @@
 # statistics
 statistics = LazyImport("import statistics")
 stats = LazyImport("from scipy import stats")
+sm = LazyImport("import statsmodel.api as sm")
+
+### Time-Series Forecast
+fbprophet = LazyImport("import fbprophet")
```

Comment:
is this the typical usage?
https://facebook.github.io/prophet/docs/quick_start.html

Looks more like "Prophet" to me?

Reply:
If I use `from fbprophet import Prophet` I'll miss out on other submodules like fbprophet.diagnostics and fbprophet.plot. If you still think that we should go with only Prophet, please do let me know. :)

Reply:
Good point.
Some variations that I am pondering:
1) I am wondering if we should add fbprophet at all then because I am not so sure how big the added value is if the user then always has to use fbprophet.plot etc.
2) We can just use fbprophet
3) We could add fbprophet and Prophet (and skip the submodules like diagnostics and plot whose names are too generic)

For now, I most prefer 3)
What do you think?

Reply:
Thanks. I also think that 3 is the optimal solution. I have made the changes as per point no. 3. :)